src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -185,7 +185,7 @@ public String FullName
             private set
             {
                 if (value == null)
-                    throw new ArgumentNullException("FullName");
+                    throw new ArgumentNullException(nameof(FullName));
```

Comment:
It was `"FullName"` before, but this should probably be corrected to `nameof(value)`.